controller/DonationController.js

```diff
@@ -45,10 +45,16 @@ exports.handleFluttwerwaveRequests = catchAsyncError(async (req, res) => {
     await Donation.createDonation(payload);
   }
 
-  Mail.sendThankYouForDonation(
-    payload.data.customer.email,
-    payload.data.amount
-  );
+  try {
+    if (payload.data.customer.name.toLowerCase() !== "anonymous") {
+      Mail.sendThankYouForDonation(
```

Comment:
Also, can we pass the donor's name to this function as an argument so that when we are saying thank you in the email, we will be addressing the donor's name

```
Mail.sendThankYouForDonation(
  payload?.data?.customer?.name,
  payload?.data?.customer?.email,
  payload?.data?.amount
)
```

Reply:
Working on this right now. Changes should be pushed in <5 minutes.